Kernel/Process.cpp

```diff
@@ -687,7 +688,41 @@ int Process::do_exec(String path, Vector<String> arguments, Vector<String> envir
         // Okay, here comes the sleight of hand, pay close attention..
         auto old_regions = move(m_regions);
         m_regions.append(move(executable_region));
+
+        AK::ArmedScopeGuard rollback_regions_guard([&](){
```

Comment:
No need to qualify with `AK::` here since you've already `using`ed it in the header.

Reply:
True. Removed the qualification.